utils/pip/pipdeptree/scriptCreator.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"github.com/jfrog/jfrog-client-go/utils/io/fileutils"
+	"io/ioutil"
+	"os"
+	"path"
+	"strings"
+)
+
+const (
+	pipDepTreeVersion             = "3"
+	pipDepTreeContentFileName     = "deptreescript.go"
+	pipDepTreePythonScript        = "pipdeptree.py"
+	pipDepTreeContentRelativePath = "utils/pip"
+)
+
+// Run manually  to generates new pipDepTreeContentFileName from pipDepTreePythonScript if there is new one and update pipDepTreeVersion number
```

Comment:
// Run manually  to generates new pipDepTreeContentFileName from pipDepTreePythonScript if there is new one and update pipDepTreeVersion number
-->
// This main function should be executed manually following changes in `pipdeptree.py`. Running the function generates new 'pipDepTreeContentFileName' from 'pipDepTreePythonScript.
// Make sure to increment the value of the 'pipDepTreeVersion' constant before running this main function.